db/migrate/20231122203839_add_name_to_categories.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToCategories < ActiveRecord::Migration[7.0]
+  def change
+    add_column :categories, :name, :string
+  end
+end
```

Comment:
- [ ] [REQUIRED] While trying to fix the previous recommendations by the previous reviewer, there are new issues that have sprung up with the migration.
- The error message indicates that there is an attempt to add a column named "name" to the "categories" table in your database, but the column "name" already exists in that table. This results in a duplicate column error.

### To resolve this
- Manually delete this particular migration file and then run a 

``` ruby
rails db:rollback
```
Afterwards, you can try to migrate again.
<img width="726" alt="Screenshot 2023-12-02 at 20 07 16" src="https://github.com/diarisdiakite/my-account/assets/58692590/4b1b4a12-dcb1-40a3-955c-d5841aa61b6b">